src/components/qas/questions/qaMultiSelect/index.jsx

```diff
@@ -0,0 +1,197 @@
+    };
+  }
+
+  componentDidUpdate() {
+    const { answerable } = this.state;
+    const { question, activeId, ids } = this.props;
+    const { id } = question;
+    const active = ids ? checkIds(ids, activeId) : activeId === id;
+
+    this.checkAnswerable(answerable, active);
+  }
+
+  checkAnswerable(answerable, active) {
+    if (!answerable && active) {
+      this.setState(prevState => ({
+        ...prevState,
+        answerable: true,
+      }));
+    }
+  }
+
+  handleChange = e => {
+    const {
+      question,
+      handleAnswerSelect,
+      focusCallback,
+      saveHandler,
+    } = this.props;
+    if (e) {
+      this.setState(
+        prevState => ({
+          ...prevState,
+          hasFocus: true,
+        }),
+        () => {
+          if (focusCallback) {
+            const { hasFocus } = this.state;
+            focusCallback(hasFocus);
+          }
+        }
+      );
+    }
+    if (!e) {
+      this.setState(
+        prevState => ({
+          ...prevState,
+          hasFocus: false,
+        }),
+        () => {
+          if (focusCallback) {
+            const { hasFocus } = this.state;
+            focusCallback(hasFocus);
+          }
+          const { selectedOptions } = this.state;
+          handleAnswerSelect(question.id, selectedOptions, e.type);
+          saveHandler();
+        }
+      );
+    }
+  };
+
+  handleStoredOptions = e => {
+    const { checked, value } = e.target;
+    if ((checked || value) !== undefined) {
+      const { selectedOptions } = this.state;
+      if (!checked) {
+        const index = selectedOptions.indexOf(value);
+        selectedOptions.splice(index, 1);
+      } else {
+        selectedOptions.push(value);
+      }
+      this.setState(prevState => ({
+        ...prevState,
+        selectedOptions,
+        selectedValue: selectedOptions.join(', '),
+      }));
+    }
+  };
+
+  getMultiSelectOptions = options => {
+    const { selectedOptions } = this.state;
+    return options.map((option, i) => {
+      return (
+        <SelectionControl
+          id={`qa-multi-select--${option.value}`}
+          checkedCheckboxIcon={<CheckBox />}
+          uncheckedCheckboxIcon={<CheckBoxOutlineBlank />}
+          // eslint-disable-next-line react/no-array-index-key
+          key={`${option.label}-${i}--option`}
+          name="selectedOptions[]"
+          label={option.label}
+          type="checkbox"
+          defaultChecked={selectedOptions.indexOf(option.value) > -1}
+          value={option.value}
+          onClick={this.handleStoredOptions}
+        />
+      );
+    });
+  };
+
+  render() {
+    const { ids, question, activeId, answer } = this.props;
+    const {
+      id,
+      label,
+      labelPre,
+      labelPost,
+      options,
+      placeholder,
+      questionType,
+    } = question;
+
+    const active = ids ? checkIds(ids, activeId) : activeId === id;
+    const { hasFocus, answerable, selectedOptions } = this.state;
```

Comment:
The toggle that opens this menu should be disabled (either literally or as close an approximation as we can muster) is answerable is false

Reply:
I'm not sure I follow you on this. I am a little confused as to what you would like me to do with this.

Reply:
for now, handling this in the CSS